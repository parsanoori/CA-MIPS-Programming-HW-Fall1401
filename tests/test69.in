54
891468173
68563468
1024215294
755998506
725266360
972144454
435085621
254028471
739394330
818226381
550659499
556787149
512270276
204022013
353402431
315574534
315604951
492925543
21299551
624036294
675391827
1050607357
28114541
352801857
906342496
114604625
550199393
568875560
681045977
827598734
460285310
936152416
781970
501468878
77962382
363317090
683051828
76404265
123008016
319282019
667695468
246818727
213595322
485727831
917258815
1016457898
712836038
1062326174
10168703
389846987
483290473
963968820
602112005
940863167