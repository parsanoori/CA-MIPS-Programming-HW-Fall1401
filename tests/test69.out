781970
10168703
21299551
28114541
68563468
76404265
77962382
114604625
123008016
204022013
213595322
246818727
254028471
315574534
315604951
319282019
352801857
353402431
363317090
389846987
435085621
460285310
483290473
485727831
492925543
501468878
512270276
550199393
550659499
556787149
568875560
602112005
624036294
667695468
675391827
681045977
683051828
712836038
725266360
739394330
755998506
818226381
827598734
891468173
906342496
917258815
936152416
940863167
963968820
972144454
1016457898
1024215294
1050607357
1062326174